data/resources/ui/window.ui

```diff
@@ -5,8 +5,30 @@
     <property name="default-height">600</property>
     <property name="content">
       <object class="GtkStack" id="main_stack">
-        <property name="visible-child">login</property>
+        <property name="visible-child">loading_widget</property>
         <property name="transition-type">crossfade</property>
+        <child>
+          <object class="GtkBox" id="loading_widget">
+            <property name="orientation">vertical</property>
+            <child>
+              <object class="GtkHeaderBar"/>
+            </child>
+            <child>
+              <object class="AdwBin">
```

Comment:
I don't think this is really needed, right?

Reply:
Absolutley